### ISSUE TITLE
Add Inter

### ISSUE THREAD

**Issue by prvcy** [google/fonts#1455]
Website: http://rsms.me/inter/
Github: https://github.com/rsms/inter

**Comment by davelab6** [comment]
Thanks for the tip :)

**Comment by hmaesta** [comment]
That's a really nice one.

**Comment by pharan** [comment]
I saw this font trend on reddit the other day. I looked at the specimen and it's really amazing.
It's designed as a for-screen body font. And a good fit for many designs that want to be pretty neutral.
Excellent documentation too.

**Comment by EliotSlevin** [comment]
I'd love for this to be in google fonts so I can use it in google slides. We're using inter-ui for branding, would be fantastic.

**Comment by jozefmak** [comment]
Please, do.

**Comment by alentodorov** [comment]
I would second the addition of Inter UI to Google Fonts as it allows usage in AMP pages as well.

**Comment by SimonFricker** [comment]
+1

**Comment by amio** [comment]
@davelab6 Is there something blocking this to next stage in the process?

BTW, Inter UI is [going to release a new version](https://github.com/rsms/inter/releases), maybe we should hold this until 3.0 released? @rsms

**Comment by jhpratt** [comment]
Would love to see this addition — would a pull request be sufficient?

**Comment by SimonFricker** [comment]
@jhpratt why not give it a go and find out.

**Comment by jhpratt** [comment]
@SimonFricker I certainly would; looking at the way current fonts are in there, it seems quite simple. The only problem I have is that there some PRs open that were created nearly two years ago. It looks like they're waiting on variable fonts, but that's now supported in all major browsers.

**Comment by PickJBennett** [comment]
I would love to see this on Google Fonts, especially the variable font!! 🤩

**Comment by luismqueral** [comment]
+1

**Comment by timc3** [comment]
+1 here as well.

**Comment by SimonFricker** [comment]
Inter UI v3.2 was released this 4 days ago.

**Comment by surjithctly** [comment]
_+1_ **Any update on this? Why members are silent?** _+1_

**Comment by ytx21cn** [comment]
I really appreciate this typeface. It has almost equivalent quality compared with Apple's San Francisco and Google's Roboto typefaces, and it is also another good choice of screen based designs.
Newest release is 3.2: https://github.com/rsms/inter/releases

**Comment by rsms** [comment]
It’s happening.

**Comment by luismqueral** [comment]
@rsms congrats, well deserved!

**Comment by sladen** [comment]
For the record; SIL OFL-1.1:

* https://github.com/rsms/inter/blob/v3.3/LICENSE.txt

**Comment by sladen** [comment]
Further status update from this week; project has been renamed `Inter UI`→`Inter`, along with an accompanying update of the Copyright message and metadata:

* https://github.com/rsms/inter/commit/5a5be63d6e59d5c437b270a5522b370ba845bf39#diff-61e0bdf7e1b43c5c93d9488b22e04170

**Comment by tdupre** [comment]
Very interested by the perspective to have Inter in Google Fonts, looking forward to the implem.! 
Thanks

**Comment by kepano** [comment]
Is there a rough ETA on when this typeface will be available on Google Fonts?

**Comment by abhinayar** [comment]
Would love to see this -- bumping!

Sincerely,
Abhi
Technical Lead | tonebase

**Comment by hugolantaume** [comment]
+1

**Comment by sanderbrauwers** [comment]
bump

**Comment by jakesyl** [comment]
Really looking forward to this one!

**Comment by nonoesp** [comment]
Great to hear this is happening. 🍾

**Comment by blaindy** [comment]
Any ETA?

**Comment by ytx21cn** [comment]
> Any ETA?

There are still some issues to be fixed as seen in rsms/inter.

**Comment by tdupre** [comment]
so is Google waiting for ALL the issues to be fixed or? This font was "high priority" on Feb 5th :) 
Any info would be appreciated! Thanks

**Comment by rsms** [comment]
> > Any ETA?
> 
> There are still some [issues](https://github.com/rsms/inter/issues) to be fixed.

No, this is incorrect.

**Comment by hmaesta** [comment]
Come on, guys. ✋🏻
We all like this font, but this kind of pressure to make it available on Google Fonts is kind of annoying, specially for the ones working to make that happen. 
While it's not available on GFonts, you can host it yourself – or work to make it available thru jsDelivr. It's not that painful.

**Comment by kepano** [comment]
> Come on, guys. ✋🏻
> We all like this font, but this kind of pressure to make it available on Google Fonts is kind of annoying, specially for the ones working to make that happen.
> While it's not available on GFonts, you can host it yourself – or work to make it available thru jsDelivr. It's not that painful.

Unfortunately if you are using Google products such as Google Docs, Slides, etc, only the fonts in Google Fonts can be used. Would love it if we could use custom fonts within Google products, but that's a different issue entirely.

**Comment by ytx21cn** [comment]
> > > Any ETA?
> > 
> > 
> > There are still some [issues](https://github.com/rsms/inter/issues) to be fixed.
> 
> No, this is incorrect.

Have you fixed existing Microsoft Word display issues?

**Comment by thundernixon** [comment]
> existing Microsoft Word display issues

These are issues in Windows but luckily are fixed in upcoming releases of Windows. I've documented this in issues in the Inter repo and in the inter google/font PR. In legacy software, people may need to use the static versions of fonts.

**Comment by efstajas** [comment]
Hey all! I'm wondering if there is anything that can be done to help with adding Inter. I'm not sure what what issues would need to be fixed for Inter that we could potentially contribute to. We've updated our CI for Grover.com to use Inter a while ago and are patiently waiting to update our products since in order to avoid self-hosting and not being able to use it on G Suite.

**Comment by theochampion** [comment]
Any updates on this guys?

**Comment by Erioldoesdesign** [comment]
Flagging that I would very much like this to be included as it's a brand font for a project and I want to use it across my google docs please!

**Comment by Uranbold** [comment]
ETA?

**Comment by johannjenson** [comment]
Does anyone have a rough ETA to share?

**Comment by marciplan** [comment]
Out of curiosity -- What is keeping such a font from appearing in Google Fonts? If I can believe the comments in here by the team behind Google Fonts then it's been a high priority for half a year now?

**Comment by ahmadajmi** [comment]
+1

**Comment by m4rc1e** [comment]
https://github.com/google/fonts/pull/1908#issuecomment-538769823

**Comment by efstajas** [comment]
heya! Over here at Grover we made the decision to switch to Inter as our main typeface almost a year ago, and have decided to wait for it to be on Google Fonts for our full rollout. At this point, we're considering potentially hiring a freelancer to help out with the process. Would this be feasible? Could someone we hire actually help out? What competence should we look for?

**Comment by luismqueral** [comment]
@efstajas I'm not exactly sure what you're asking, but in the meantime you can host the font yourself.

**Comment by m4rc1e** [comment]
@thundernixon Are the static fonts ok? it may be best to leave the vfs till next year and just give the masses what they want.

**Comment by efstajas** [comment]
> I'm not exactly sure what you're asking, but in the meantime you can host the font yourself.

Of course we could, but it's more about the fact that we wouldn't want to make the switch before all of our teams can use a font in presentations and documents in Google Docs, for example. Anyway, what I'm asking is — since this is an open source project, is there any opportunity for us to contribute by hiring someone to work on it? I'm just trying to find out if any kind of external help could speed things up or not.

**Comment by fuzenco** [comment]
Just today I went looking for this font in Google Fonts. Couldn’t believe it wasn't there. Please make this happen Google.

**Comment by jshamg** [comment]
why the heck is this font not in Google Fonts? What are you guys doing?

**Comment by davelab6** [comment]
@thundernixon I think it could be good to make a concrete list of the pending tasks you see until this is ready to PR

**Comment by thundernixon** [comment]
list of pending tasks added to the PR: https://github.com/google/fonts/pull/1908#issuecomment-573878839

**Comment by boomtemp123** [comment]
This appears to have passed a few days ago, when can we expect to see it included?

**Comment by adamgreenough** [comment]
Almost 2 years later... any sign of this appearing on G Fonts soon? 😦

**Comment by bailsafe** [comment]
Looks like the checks have passed on pull #1908, hoping to see it on G Fonts soon. Hoping for an update.

**Comment by jshamg** [comment]
I really hope so, too

**Comment by rickyzhangca** [comment]
this indeed took pretty long

**Comment by hmaesta** [comment]
![](https://media.giphy.com/media/vv41HlvfogHAY/giphy.gif)

**Comment by efstajas** [comment]
wohoo! gonna pop a bottle of champagne with the design team tonight. let's hope it makes it to prod soon 🥳

Thanks everyone!!

**Comment by EliotSlevin** [comment]
Yay!

**Comment by kuvahaku** [comment]
❤︎

**Comment by surjithctly** [comment]
I initially commented on Jan 30, 2019. Now after a year, Its now merged. ❤︎❤︎❤︎

How long it will take to appear on the google fonts website?

**Comment by mattdell** [comment]
There's some explanation here: https://github.com/google/fonts/pull/1908#issuecomment-580509162

> As with any other typeface in here, internal Google Fonts folks will ultimately decide when and how to bring this into production, and it may be held back temporarily. A current obstacle is that no modern browsers currently properly support the font-style properly as specified by the CSS Level 4 Fonts Module for variable fonts with a slnt axis, like Inter. For a full explanation and test case of this, plus links to filed browser bugs, please see https://arrowtype.github.io/vf-slnt-test/.

**Comment by efstajas** [comment]
Is there any way to follow the process from now on? Not trying to be pushy or anything, just curious!

**Comment by cjlaborde** [comment]
Would like to know as well, wanted this font for a long time now.

**Comment by m4rc1e** [comment]
Hopefully this week.

**Comment by brampitoyo** [comment]
You’d be pleased to know that Inter has been added to Google Fonts today: https://fonts.google.com/specimen/Inter

**Comment by jshamg** [comment]
Time to celebrate 🍾🎉🎊

**Comment by yasendimov** [comment]
This has been quite the ride. Thanks guys!

**Comment by luismqueral** [comment]
What a time to be alive.

**Comment by Erioldoesdesign** [comment]
Thank you! huzzah I can use Inter in my google stuff now

**Comment by patrickcate** [comment]
Is there a separate issue for adding the italic styles?

**Comment by ytx21cn** [comment]
Thanks for the effort!

**Comment by kepano** [comment]
I don't see Inter available in Google Docs/Slides yet, is that a caching thing or is there another way that Google Fonts make their way to those apps?

**Comment by shstkvch** [comment]
Congatulations! Very glad this has been added in

**Comment by brampitoyo** [comment]
Inter is now available for use in Google Docs, Sheets and Slides.

![](https://user-images.githubusercontent.com/51007/76799730-fa97da80-6836-11ea-9ab7-19bf5d9aaabc.png)

**Comment by nik72619c** [comment]
Hi,
Could anyone guide me how to find the inter variable font file ?
Thanks!

**Comment by jshamg** [comment]
@nik72619c on his Website [https://rsms.me/inter/](https://rsms.me/inter/).
